assemblyscript/mandelbrot-asc.js

```diff
@@ -30,9 +30,7 @@ module.exports = async () => {
       if (!imgData) imgData = ctx.createImageData(WIDTH, HEIGHT);
       const offset = getDataBuffer();
       const linearMemory = new Uint8Array(memory.buffer, offset, WIDTH * HEIGHT * 4);
-      for (let i = 0, len = linearMemory.length; i < len; i++) {
-        imgData.data[i] = linearMemory[i];
-      }
+      imgData.data.set(linearMemory);
```

Comment:
Well spotted - didn't realise there was a `set` method, there are a few other projects I could use that in.